backend/static/js/qleverUI.js

```diff
@@ -567,7 +617,9 @@ async function processQuery(sendLimit=0, element=$("#exebtn")) {
         scrollTop: $("#resTable").scrollTop() + 500
       }, 500);
       
-      appendRuntimeInformation(result.runtimeInformation, result.query, result.time);
+      // MAX_VALUE ensures this always has priority over the websocket updates
+      appendRuntimeInformation(result.runtimeInformation, result.query, result.time, { queryId, updateTimeStamp: Number.MAX_VALUE });
+      visualise(false);
```

Comment:
Also we need to close the socket here if it's not already closed to prevent it from waiting forever